docs/content/en/docs/clustermgmt/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -31,6 +31,11 @@ If you have a management cluster running EKS Anywhere version v0.15, you can suc
 When triggering a workload cluster upgrade after upgrading the management cluster, please keep in mind that it will not only apply your changes in the workload cluster spec, but also any new improvements included in the new EKS Anywhere controller that was upgraded on the management cluster.
 The changes in the EKS Anywhere controller can trigger a machine rollout on the workload cluster during upgrade, even if the changes to the workload cluster spec didn't require one (for example, scaling down a worker node group).
 
+To upgrade a cluster to v0.18.x `eksctl anywhere` version, it is madatory to upgrade the cluster first to `eksctl anywhere` version v0.17.x. 
+
+From `eksctl anywhere` version v0.18.0 onwards, for the management cluster upgrade, the CLI waits for 1 hour combined for etcd to be ready, control plane to be ready, default CNI to be configured, worker nodes to be ready and cluster to be ready state. It is recommended to run the `upgrade` command with the `--no-timeouts` flag if you want to override 1 hour timeout behavior.
```

Comment:
1. We wanted to make this visible so if product thinks that’s not enough, we can change it. 
2. Since this section just includes [important data points](https://anywhere.eks.amazonaws.com/docs/clustermgmt/cluster-upgrades/baremetal-upgrades/), I have added it when we explain upgrade process more in [detail](https://anywhere.eks.amazonaws.com/docs/clustermgmt/cluster-upgrades/baremetal-upgrades/#upgrade-cluster-command).

Reply:
@panktishah26 What point are we trying to convey though? The paragraph starts with 'v0.18 onwards' and talks about the CLI having a 1h timeout but that's not new to 0.18, hence I'm a little confused.

@csplinter

Reply:
Discussed this with Chris offline, I have removed this line for now. We can add it later anytime if we want to specify this.